File: Recursive/5.py
def towersOfHanoi(n, start, end, sub):
    if n == 1:
        print("Move disk 1 from rod", start, "to rod", end)
        return
    else:
        towersOfHanoi(n-1, start, end, sub)
        print("Move disk", n, "from rod", start, "to rod", end)
        towersOfHanoi(n-1, sub, start, end)


towersOfHanoi(3, 'A', 'C', 'B')

File: Recursive/5.c
#include <stdio.h>
void towersOfHanoi(int N, char start, char end, char sub){
    if (N==1){
        printf("%d disk -> %c %c\n", N,start,end);
        return;
    }
    else{
        towersOfHanoi(N-1,start,sub,end);
        printf("%d disk -> %c %c\n",N,start,end);
        towersOfHanoi(N-1,sub,end,start);
    }
}
int main(){
    int N;
    scanf("%d",&N);
    towersOfHanoi(N,'A','C','B');
}